packages/extension-api/src/extension-api.d.ts

```diff
@@ -2140,6 +2140,7 @@ declare module '@podman-desktop/api' {
     OpenStdin?: boolean;
     StdinOnce?: boolean;
     Detach?: boolean;
+    startOnCreation?: boolean;
```

Comment:
sorry, but maybe we could just have `start` as we're inside the create Operation